.github/workflows/ci-testing-deploy.yml

```diff
@@ -1721,9 +1721,87 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-node-
       - name: setup
-        run: ./ci/github/system-testing/e2e.bash install
+        run: ./ci/github/system-testing/e2e.bash install_sleepers
       - name: test
-        run: ./ci/github/system-testing/e2e.bash test
+        run: ./ci/github/system-testing/e2e.bash test_sleepers
+      - name: recover docker logs
+        id: logs_recovery
+        if: always()
+        run: ./ci/github/system-testing/e2e.bash dump_docker_logs
+      - name: upload docker logs
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ github.job }}_docker_logs
+          path: simcore_logs
+      - name: upload screenshots
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ github.job }}_screenshots
+          path: tests/e2e/screenshots
+      - name: upload e2e logs
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ github.job }}_logs
+          path: tests/e2e/logs
+      - name: cleanup
+        if: always()
+        run: ./ci/github/system-testing/e2e.bash clean_up
+
+  system-test-e2e-mattward:
+    timeout-minutes: 20 # if this timeout gets too small, then split the tests
+    name: "[sys] e2e mattward"
+    needs: [build-test-images]
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python: [3.6]
+        node: [14]
+        os: [ubuntu-20.04]
```

Comment:
One question here: can't we use the matrix field to define the e2e file instead of creating a full new entry?